changelogs/unreleased/gh-8613-stop-running-repl-by-default.md

```diff
@@ -0,0 +1,3 @@
+## feature/cli
+
+* Stop running Lua REPL by default and yield the message with usage (gh-8613).
```

Comment:
It would be convenient to have the information in the initial ticket on how to achieve the old behavior (use `--repl` option?).

Reply:
Added this line:
> To run Lua REPL just set `-i` flag.